docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,172 @@
+
+There are two main types of CAs, each with a related role in generated the certificates associated with an organization.
+
+* One CA (or more, if you are using intermediate CAs) is used to generate (through a process called "enrollment") the certificates of the admin of an organization, the MSP of that organization, and any nodes owned by that organization. This CA will also generate the certificates for any additional users. Because of its role in "enrolling" identities, this CA is sometimes called the "enrollment CA" or the "ecert CA".
+* The other CA generates the certificates used to secure communications on Transport Layer Security (TLS). For this reason, this CA is often referred to as a "TLS CA". These TLS certificates are attached to actions as a way of preventing "man in the middle" attacks. Note that the TLS CA is only used for issuing certificates and can be shut down when that activity is completed. Users have the option to use one way (server only) TLS as well as two way (server and client) TLS, with the latter also known as "mutual TLS". **Because specifying that your network will be using TLS (which is recommended) should be decided before deploying the "enrollment" CA (the YAML file specifying the configuration of this CA has a field for enabling TLS), you should deploy your TLS CA first and use it to generate a certificate that will be used when bootstrapping your enrollment CA.** Note that organizations can decide to share a TLS CA for the generation of TLS certificates if they wish. However, this increases the exposure of the TLS CA in case. If all of the organizations in a network use a single TLS CA and this CA is compromised, it might not be possible to change the configuration of the network to enable new TLS certificates to be used on the network. Note that TLS CA certs from a TLS CA known to the network can be transferred to other members out of band.
```

Comment:
Talked to Joe about this in person. When we say to enroll with the TLS CA,  depending on how the TLS CA is configured, the enrollment request to the TLS CA might require specifying which profile to enroll against. I believe this might be covered in the more in depth doc by Pam.